CONTRIBUTORS

```diff
@@ -72,6 +72,7 @@ Luis Cruz
 Marc Lasson
 Martin Fietz
 Martin Olsson
+Masoud Pezeshkzade
```

Comment:
Thanks for your PR. The contributors list is generated automatically before the next release (based on `git log`). Please do not change it manually.

Additionally, it would be great if you could use the same name for all commits, so that we do not have to filter duplicate items manually after generating the list.
![image](https://user-images.githubusercontent.com/5811634/67634848-0ce19e00-f8c1-11e9-8bcc-8f4e89247ff6.png)

Reply:
It is my first real contribute and I didn't know how should I commit. I committed two times because one of them was about the discovery list and another was about the subscription list. Please tell me what should I do now?

Reply:
I guess the third commit was done using the GitHub web interface, while the others were done using the local git client on your PC. If you select the same email address for the git client, your local commits can be linked to the GitHub user name. Additionally, we use that name for the contributors list, so if you want your full name to be listed, you need to change the author names of the commits and then force-push.

Changing existing commit's authors is described on the website below. For rebasing, I suggest to use `git rebase -i HEAD^^^`
https://makandracards.com/makandra/1717-git-change-author-of-a-commit